nepalingo-web/src/hooks/useQuotes.tsx

```diff
@@ -51,26 +57,29 @@ const useQuotes = ({ language }: QuoteProps) => {
     if (quotesText) {
       const loadQuotes = () => {
         const quotesArray = quotesText.split("\n").map((line: string) => {
-          const [text, translation] = line.split(",");
+          const [text, newari, tajpuria, maithili] = line.split(",");
           return {
             text: text ? text.trim().replace(/(^"|"$)/g, "") : "",
-            translation: translation
-              ? translation.trim().replace(/(^"|"$)/g, "")
-              : "",
+            translation: {
+              newari: newari ? newari.trim().replace(/(^"|"$)/g, "") : "",
```

Comment:
I don't think you need this .replace - what is it for?

Reply:
I think that's just to remove the `|` from the csv